src/Task04_StringAndIntDateFormatter/DateFormatter.test.js

```diff
@@ -1,46 +1,48 @@
-const DateFormatter = require("./DateFormatter");
+import DateFormatter from './DateFormatter.js'
 
-describe("DateFormatter Tests", () => {
+describe('DateFormatter Tests', () => {
   test('formats "31102011" to "31-10-2011"', () => {
-    const formatter = new DateFormatter("31102011");
-    expect(formatter.format()).toBe("31-10-2011");
-  });
+    const formatter = new DateFormatter('31102011')
+    expect(formatter.format()).toBe('31-10-2011')
+  })
 
   test('formats "31102011" to "31 October 2011"', () => {
-    const formatter = new DateFormatter("31102011");
-    expect(formatter.format("DD Month YYYY")).toBe("31 октомври 2011");
-  });
+    const formatter = new DateFormatter('31102011')
+    expect(formatter.format('DD Month YYYY')).toBe('31 октомври 2011')
+  })
 
   // JavaScript Data object corrects incorrect dates like April 31st
   test('formats "20130431" (YYYYMMDD) to "31 April 2013"', () => {
```

Comment:
test description says 31st of Aipril, but expected is 1st of May. I guess something is not working properly